pkg/ingress/ingress.go

```diff
@@ -52,12 +52,20 @@ func InsertProbe(ing *v1alpha1.Ingress) (string, error) {
 		if rule.HTTP == nil {
 			return "", fmt.Errorf("rule is missing HTTP block: %+v", rule)
 		}
+		probePaths := make([]v1alpha1.HTTPIngressPath, 0, len(rule.HTTP.Paths))
 		for i := range rule.HTTP.Paths {
```

Comment:
kind of surprised nothing needed to change in ingress_test.go for this.. is it missing a test?

Reply:
Yeah, I'll check it out this morning.  I was mainly focused on fixing unit tests and whether it blew up any of the kingress using it.  I think currently net-contour is the only implementation that doesn't "just" auto-import this, which is sort of cool.

Reply:
ok, the test is written in a funny way.  I expanded to cover this.